spec/acceptance/validate_ruby_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'Running ruby validation' do
+describe 'pdk validate ruby', module_command: true do
```

Comment:
Agreed on the magic thing. And I can't say I'm blown away by the rspec docs either :-( It's definitely something we can keep an eye on, and easy enough to change our minds afterwards.

The main advantage is in my view that it allows for external composeability, that is, a new shared context can add itself to `module_commands` without requiring changes across the entire test suite. Admittedly that is pretty weak sauce, and also [YAGNI](http://wiki.c2.com/?YouArentGonnaNeedIt).